astropy/utils/iers/tests/test_leap_second.py

```diff
@@ -13,6 +13,63 @@
 from astropy.utils.iers import iers
 from astropy.utils.data import get_pkg_data_filename
 
+# Import every top-level astropy subpackage as a test that the ERFA leap second
```

Comment:
How about putting this in a new test file? E.g., `astropy/tests/tests` has a `test_imports.py` and one could add an `test_no_net_on_import.py` (or better name...) there. Then, if someone does something like what I did for the leap seconds, they'll see a failure in a file that will hopefully make sense.

Reply:
I actually did this initially. The problem I ran into is that the current `test_leap_second.py` requires the modified `erfa.leap_seconds` to pass, whereas the new imports-only test file would require the unmodified version to pass. So the success of tests passing depends on the order of test collection. 

The only ways I could see around this are:
1. Put things in one file where the sequence of testing `erfa.leap_seconds` and then updating it is controlled, or
1. Make some of the tests here have some kind of setup that does the leap second update. IIRC there are 3 tests here that fail without it, so I guess that is an option, but the first option seemed a bit more straightforward.

Reply:
That makes sense. One option to at least make this shorter is to do
```
from astropy.tests.tests import test_imports

test_imports()
# Instead of all the explicit imports
```